dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -58,21 +56,26 @@ PreparedStatement buildInsertPreparedStatement(String tableName, DbParameterAcce
      */
 
     /**
-     * Create a {@link PreparedStatement} object and binds fixture symbols to
+     * Create a {@link StatementExecution} and binds fixture symbols to
```

Comment:
Should we apply this `{@link StatementExecution}` to the Javadoc for the other methods?